src/index.ts

```diff
@@ -49,14 +49,18 @@ const getValidationContext = (validation: t.ValidationError) =>
   // https://github.com/gcanti/fp-ts/pull/544/files
   validation.context as t.ContextEntry[];
 
+export const TYPE_MAX_LEN = 160; // Two lines of 80-col text
```

Comment:
[Req]: This solves the CI error. It is for the generated documentation (using [docs-ts](https://github.com/gcanti/docs-ts))


```suggestion
/**
 * @category internals
 * @since 1.2.1
 */
export const TYPE_MAX_LEN = 160; // Two lines of 80-col text
```